src/Microsoft.AspNet.Authentication.JwtBearer/JwtBearerHandler.cs

```diff
@@ -150,23 +151,21 @@ internal class JwtBearerHandler : AuthenticationHandler<JwtBearerOptions>
                 await Options.Events.AuthenticationFailed(authenticationFailedContext);
                 if (authenticationFailedContext.HandledResponse)
                 {
-                    return authenticationFailedContext.AuthenticationTicket;
+                    return AuthenticateResult.Success(authenticationFailedContext.AuthenticationTicket);
                 }
-
                 if (authenticationFailedContext.Skipped)
                 {
-                    return null;
+                    return AuthenticateResult.Success(ticket: null);
                 }
 
                 throw;
```

Comment:
Looks like this PR doesn't solve (yet) the most important problem: making all the authentication middleware consistent when returning an error/throwing an exception and avoid surfacing exceptions inappropriately.

Sadly, this middleware will continue returning a 500 error for an expired access token :sob: